.eslintrc

```diff
@@ -6,6 +6,7 @@
     "browser": true// important so document & window are not undefined
   },
   "rules": {
+    "linebreak-style": 0,
```

Comment:
probably add 
`"linebreak-style": 0,// linebreak-style: ["error", "windows"] is better but seems to also cause errors in linux